lib/inertia_rails.rb

```diff
@@ -7,6 +7,11 @@
 require 'patches/mapper'
 
 ActionController::Renderers.add :inertia do |component, options|
+  if component.is_a?(Hash) && options[:props].nil?
+    options[:props] = component
+    component = true
+  end
```

Comment:
Any reason we couldn't/shouldn't put this logic in the renderer file? Its initialize method is already figuring out how to handle the (overloaded) component argument. Might be nice for Future Us to have the argument munging logic in one place.

Reply:
Should we check for this condition and raise a helpful error?

```ruby
render inertia: { users: User.all }, props: { just_kidding: 'about that users prop!' }
```

As it is I think this would fail as a white screen of death on the frontend?

Reply:
Added a check